Scripts/projects.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+function Project(name, url, imgUrl, body, datePublished){
+  this.name = name;
+  this.url = url;
+  this.imgUrl = imgUrl;
+  this.body = body;
+  this.datePublished = datePublished;
+}
+
+var projects = [new Project('Tutor Fellows', 'https://github.com/npisciotti1/Tutor-Fellows', 'http://placekitten.com/200/300', 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.', 'Wed Nov 30 2016'),
```

Comment:
As a heads up, beginning next week we will be converting our data into a separate file.